File: test/unit/bash_test.rb
#!/usr/bin/env ruby

$use_tree=$emit
$use_tree=false

require_relative '../parser_test_helper'

class BashTest < ParserBaseTest

  include ParserTestHelper

  def test_pipe
    parse "bash 'ls -al' | column 1"
  end

  # def test_selector
  #   parse "column 1 from ls -all"
  # end
end

File: angelos/test/bash_test.py
import _global
_global.use_tree = _global.emit
_global.use_tree = False
from parser_test_helper import *


class BashTest(ParserBaseTest):
    

    def test_pipe(self):
        parse("bash 'ls -al' | column 1")
        parse("bash 'ls -al' | column 1")